testsuite/podman_runner/05_install_gems_in_controller.sh

```diff
@@ -2,5 +2,7 @@
 set -xe
 sudo -i podman exec controller bash -c 'export GEM_PATH="/usr/lib64/ruby/gems/3.3.0"'
 sudo -i podman exec controller bash -c "cd /testsuite && bundle.ruby3.3 install --gemfile Gemfile --verbose"
+sudo -i podman exec controller bash -c "cd /testsuite; bundle.ruby3.3 check || bundle.ruby3.3 install --gemfile Gemfile --verbose"
```

Comment:
```suggestion
sudo -i podman exec controller bash -c "cd /testsuite; bundle.ruby3.3 check || bundle.ruby3.3 clean --force; bundle.ruby3.3 cache"
```

Reply:
Can you explain your change? Why would you run the cache command instead of install and after doing a clean?

Reply:
Those commands are cleaning the cache, and re-doing again all the cache.
And by your description, you said that by cleaning the cache the problem dissapear, so that's why I suggested it.

Reply:
Sure. I tried with that and it failed :( . The clean --force returned an error that the prometheus-client gems was not installed :open_mouth:  and the cache was building a cache directory. I tried with "cache clean", but seems we have an old version of bundler/ruby.